opae.spec.fedora

```diff
@@ -13,11 +13,9 @@ Requires:       uuid
 Requires:	json-c
 Requires:	python3
 Requires:	systemd
-Requires:	libcap
 Requires:	tbb
 Requires:	hwloc
 Requires:	spdlog
-Requires:	libedit
```

Comment:
We still need this.  It is used by opae.io.

Reply:
RPMlint errors
removed from Requires
opae.x86_64: E: explicit-lib-dependency libcap
opae.x86_64: E: explicit-lib-dependency libedit


BuildRequires:  libedit-devel

Reply:
We still need this.  It is used by opae.io.